content/03.methods.md

```diff
@@ -77,7 +77,8 @@ We required at least 60% of RNA-Seq reads mapped to the human reference or sampl
 #### SNV and INDEL calling
 
 We used four variant callers to call SNVs and INDELS from targeted DNA panel, WXS, and WGS data: Strelka2, Mutect2, Lancet, and VarDict.
-The same input interval BED files were used for both panel and WXS data.
+The same input interval BED files were used for both panel and WXS data and these were padded by 100 bp for all variant calling algorithm runs.
+The BED files for WGS were not padded for Mutect2 and Strelka2 runs, were padded by 300 bp for Lancet, and by 100 bp for VarDict.
```

Comment:
Why were these padding window numbers chosen? Also, I have the same question about how this window padding is reported.

Reply:
whoops, answered this above. Does that help?

Reply:
Yes! It does. Can you add this information in the text?

Reply:
done!